FreeRTOS/Demo/CORTEX_MPU_M3_MPS2_QEMU_GCC/FreeRTOSConfig.h

```diff
@@ -68,9 +68,10 @@ extern void vAssertCalled( void );
 #define configUSE_COUNTING_SEMAPHORES                    1
 #define configSUPPORT_DYNAMIC_ALLOCATION                 1
 #define configSUPPORT_STATIC_ALLOCATION                  1
-#define  configNUM_TX_DESCRIPTORS                        15
+#define configNUM_TX_DESCRIPTORS                         15
 #define configSTREAM_BUFFER_TRIGGER_LEVEL_TEST_MARGIN    2
 #define configCHECK_FOR_STACK_OVERFLOW                   2
+#define configALLOW_UNPRIVILEGED_CRITICAL_SECTIONS      1
```

Comment:
Why do we need it as 1 and not 0? We recommend 0.

Reply:
i'll fix that, it was from another demo, i didn't go through the list